src/myuw-notifications.js

```diff
@@ -188,6 +190,21 @@ export class MyUWNotifications extends HTMLElement {
       }
     }
 
+    /**
+     * Close menu if current focused (tabbed) element is outside the component
+     */
+    autoCloseMenu(event) {
+      this.focusableElements = !this.focusableElements ? this.getFocusableElements() : this.focusableElements;
```

Comment:
Readability. The conditional here is negated. "If not focusableElements getFocusableElements(). Otherwise this.focusableElements." Could a not-negated condition accomplish the same thing with better readability? "If focusableElements, focusableElements. Otherwise, getFocusableElements()."